### PR TITLE
Simplify `omf cd` code

### PR DESCRIPTION
**Pull request by bpinto**
This also namespace the function to `omf.packages`.

### CODE REVIEW

File: pkg/omf/cli/omf.packages.cd.fish [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# SYNOPSIS
+#   Change the current working directory to plugin / theme home directory.
+#
+# OPTIONS
+#   <name> Name of the package or theme.
+
+function omf.packages.cd -a name -d "Change the current working directory to plugin / theme home directory."
+  if test -z $name
```
Comment (derekstavis) [inline pkg/omf/cli/omf.packages.cd.fish]: `"$name"`

File: pkg/omf/cli/omf.packages.cd.fish [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    pushd $OMF_PATH
+    return 0
+  end
+
+  for path in {$OMF_PATH,$OMF_CONFIG}/{pkg,themes}/$name
+    if test -e $path
```
Comment (derekstavis) [inline pkg/omf/cli/omf.packages.cd.fish]:
```fish
if test -e "$path"
```

File: pkg/omf/cli/omf.packages.cd.fish [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# OPTIONS
+#   <name> Name of the package or theme.
+
+function omf.packages.cd -a name -d "Change the current working directory to plugin / theme home directory."
+  if test -z "$name"
+    pushd $OMF_PATH
+    return 0
+  end
+
+  for path in {$OMF_PATH,$OMF_CONFIG}/{pkg,themes}/$name
```
Comment (derekstavis) [inline pkg/omf/cli/omf.packages.cd.fish]:
```fish
for path in {$OMF_CONFIG,$OMF_PATH}/{pkg,themes}/$name
```

Comment (bpinto) [inline pkg/omf/cli/omf.packages.cd.fish]: This is actually a missing feature on `omf cd`... But I'm fine with having the custom one coming first.

File: pkg/omf/omf.fish [hunk 4]
```diff
@@ -55,9 +55,9 @@ function omf -d "Oh My Fish"
     case "c" "cd"
       switch (count $argv)
         case 1
-          omf.cd
+          omf.packages.cd
```
Comment (derekstavis) [inline pkg/omf/omf.fish]:
```fish
omf.packages.cd omf```

File: pkg/omf/cli/omf.packages.cd.fish [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# SYNOPSIS
+#   Change the current working directory to plugin / theme home directory.
+#
+# OPTIONS
+#   <name> Name of the package or theme.
+
+function omf.packages.cd -a name -d "Change the current working directory to plugin / theme home directory."
+  if test -z "$name"
```
Comment (derekstavis) [inline pkg/omf/cli/omf.packages.cd.fish]: Uneeded if `omf.packages.cd` is called with package `omf`.

File: pkg/omf/omf.fish (hunk 4 above)
Comment (bpinto) [inline pkg/omf/omf.fish]: That's not the same thing, I want to go to `omf root folder`, not to the plugin named `omf`. :)

File: pkg/omf/cli/omf.packages.cd.fish (hunk 5 above)
Comment (bpinto) [inline pkg/omf/cli/omf.packages.cd.fish]: See message above.

File: pkg/omf/omf.fish (hunk 4 above)
Comment (derekstavis) [inline pkg/omf/omf.fish]: So better pull that code from `omf.packages.cd` and bring up here, it's more explicit, and as `$OMF_PATH` isn't a plugin more concise too.

**Comment by derekstavis** [comment]
LGTM :+1:

**Comment by derekstavis** [comment]
The only missing bit with `omf cd` is that completions are... well... incomplete. This is a inherited bug.

**Pull request merged**